ddtrace/_trace/utils_botocore/span_tags.py

```diff
@@ -31,6 +35,13 @@ def set_botocore_patched_api_call_span_tags(span: Span, instance, args, params,
         if params and not config.botocore["tag_no_params"]:
             aws._add_api_param_span_tags(span, endpoint_name, params)
 
+        if (
+            config.botocore["payload_tagging_request"]
+            and config.botocore["payload_tagging_request"].replace(" ", "")
```

Comment:
nit: Could we maybe do this clean up in `patch.py`?

Reply:
I actually don't think that trimming out whitespace is needed anymore so I'll opt to just remove it.